### ISSUE TITLE
config error in JWK code

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/my-learning-analytics#1153]
With LTI enabled, in the url `/lti/jwks` it gives the error:

```
NameError at /lti/jwks/
name 'check_if_success_getting_tool_config' is not defined
```

We should probably either remove this route or fix this method so this it exists.

https://github.com/tl-its-umich-edu/my-learning-analytics/blob/b66eaa21027c0de38a1df0c9d3a0a7786e0d600a/dashboard/lti_new.py#L105

**Comment by jonespm** [comment]
It looks like this code was removed by #1054 when `check_if_success_getting_tool_config` refactored into `get_tool_conf`. Perhaps this code was just missed?

**Comment by jonespm** [comment]
Assigning to @lsloan since he was the last to change this code

**Comment by lsloan** [comment]
I accidentally allowed conflicts to creep into my branches between working on #1065 and #1054.  When I attempted to resolve the conflicts, I made a mistake and allowed this old reference to reappear.  I'm not sure how I let that reference into the code.  I'm sure it had been working on my computer before I committed, but perhaps my server was running old code at the time when I thought it was up to date.  I'm also surprised this mistake made it through code review.
